draft-ietf-tls-tls13.md

```diff
@@ -3071,21 +3230,26 @@ MUST check any candidate cipher suites against the "signature_algorithms"
 extension before selecting them. This is somewhat inelegant but is a compromise
 designed to minimize changes to the original cipher suite design.
 
+> If sent by a server, the hash and signature algorithms used in the
+signature MUST be one of those present in the
+supported_signature_algorithms field of the CertificateRequest
+message.
+
 > In addition, the hash and signature algorithms MUST be compatible with the key
-in the server's end-entity certificate. RSA keys MAY be used with any permitted
+in the sender's end-entity certificate. RSA keys MAY be used with any permitted
 hash algorithm, subject to restrictions in the certificate, if any.
 RSA signatures MUST be based on RSASSA-PSS, regardless of whether
 RSASSA-PKCS-v1_5 appears in "signature_algorithms".
 SHA-1 MUST NOT be used in any signatures in CertificateVerify,
 regardless of whether SHA-1 appears in "signature_algorithms".
 
-###  Server Finished
+####  Finished
 
 When this message will be sent:
 
-> The Server's Finished message is the final message sent by the
-server and is essential for providing authentication of the server
-side of the handshake and computed keys.
+> The Finished message is the final message in the authentication
+block. It is server and is essential for providing authentication of the
+the handshake and of the computed keys.
```

Comment:
broken grammar and double "the" here

I suggest you just change this to one sentence:

"The Finished message is the final message in the authentication block, and is essential for providing authentication of the handshake and computed keys."